.github/workflows/build.yml

```diff
@@ -89,13 +89,46 @@ jobs:
         command: test
         args: ${{matrix.flags}}
 
+#  nightly_features_build:
```

Comment:
Can we remove this as it no longer requires nightly?  Although it is already commented out...

Reply:
Yep.